VerifyReplication/storage/s3.js

```diff
@@ -98,31 +85,29 @@ function getObjMd(params, cb) {
         return cb(new Error(errMsg));
     }
 
-    return client.headObject({
-        Bucket: bucket,
-        Key: key,
-        VersionId: versionId,
-    }, (err, data) => {
-        if (err) {
-            return cb(err);
+    try {
+        const commandParams = {
+            Bucket: bucket,
+            Key: key,
+        };
+        if (versionId) {
```

Comment:
do we need this `if` ?
is it not the same as `VersionId: versionId || undefined`, or even just `VersionId: versionId` as before?